src/packages/application/index.js

```diff
@@ -30,7 +30,7 @@ class Application extends Base {
   async boot() {
     const { root, router, domain, server, port } = this;
     const store = Database.create(
-      require(`${root}/config/database.json`)
+      require(`${root}/config/database.js`)
```

Comment:
We can drop the `.js` from require. Also after refactoring to `export default` from the config files we'll have to update this to:

```javascript
const store = Database.create(
  require(`${root}/config/database`).default
);
```